### DISCUSSION TITLE
Not interested in... myself?

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#121282] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

@jidanni is me. It doesn't make too sense to not be interested in one's self, no?

![20240427T134938](https://github.com/community/community/assets/1230959/98ced981-8a27-452d-b1a0-504095568f8b)

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Comment by matt-sdc** [comment]
I would agree with above design.  I dont care about activity on repos that I did.  I already know about that activity, as.. i did it.
